libs/facilities/frontend/domain/src/lib/application/facades/browse.facade.ts

```diff
@@ -0,0 +1,50 @@
+import { inject, Injectable } from '@angular/core';
+import { faker } from '@faker-js/faker';
+import { map } from 'rxjs';
+
+import { TimeseriesRequestService } from '../../infrastructure/timeseries-request.service';
+
+/**
+ * Browse facades service.
+ */
+@Injectable({ providedIn: 'root' })
+export class XdBrowseFacade {
+	private readonly _scanService = inject(TimeseriesRequestService);
+
+	/**
+	 * Get all timeseries.
+	 */
+	public getAllTimeseries() {
+		return this._scanService.getAllTimeseries().pipe(
+			map((timeSeriesItem) => {
+				return timeSeriesItem.map((timeSeriesItem) => {
+					return {
+						id: timeSeriesItem.entityId,
+						icon: faker.helpers.arrayElement([
+							'battery-empty',
+							'water-fish',
+							'water-plant',
+							'truck',
+						]),
+						notification: `${faker.number.int({ min: 0, max: 99 })}`,
+						heading: timeSeriesItem.entityId,
+						subheading: timeSeriesItem.propertySetName,
+						variant: 'success',
+						pumps: faker.number.int({ min: 0, max: 99 }),
+						location: faker.location.city(),
```

Comment:
This is probably part of another issue, but we need to actually store the values in the backend and retrieve them and not just generate random values. To be honest I don't even know how facilities are stored in the backend and which attributes they have so for now I guess generating random stuff is better than nothing.

Reply:
Currently the frontend and backend use different interfaces when communicating, f.e. the backend currently does not provide the icon property which is needed by the frontend. 

Therefore I simply mapped the values to fit. => `I add a TODO comment that this needs to be changed`